src/Components/PokemonCard/PokemonCard.js

```diff
@@ -0,0 +1,41 @@
+import React, {useEffect, useState} from 'react';
+import axios from "axios";
+import './PokemonCard.css';
+
+
+const PokemonCard =({url}) => {
+    const [pokemonData, setPokemonData] = useState([]);
+
+    useEffect(() => {
+        async function fetchData() {
+            try {
+                const result = await axios.get(`${url}`);
+                /*console.log(result.data)*/
+                setPokemonData(result.data)
+            } catch (e) {
+                console.error(e);
+            }
+        } fetchData()
+    }, []);
+
+    return (
+        <>
+            {Object.keys(pokemonData).length > 0 && <article className="pokemon-card">
```

Comment:
Hier kan die dan simpeler
```suggestion
            {pokemonData && <article className="pokemon-card">
```